v0/destinations/mp/data/MPEventPropertiesConfig.json

```
@@ -166,5 +166,13 @@
   {
     "sourceKeys": "messageId",
     "destKey": "$insert_id"
+  },
+  {
+    "sourceKeys": "context.sessionId",
+    "destKey": "$session_id",
+    "required": false,
+    "metadata": {
+      "type": "toNumber"
+    }
```

```suggestion
    "sourceKeys": "sessionId",
    "destKey": "$session_id",
    "required": false,
    "sourceFromGenericMap":true,
    "metadata": {
      "type": "toNumber"
    }
```
We have already adding a sessionId field in GenericFieldMapping.json for amplitude destination. We can update this section after that [pr](https://github.com/rudderlabs/rudder-transformer/pull/1419#pullrequestreview-1108567504) got merged.

---

Sure, makes sense